src/effects/VST3/VST3Utils.cpp

```diff
@@ -94,86 +94,3 @@ wxString PresetsBufferStream::toString() const
 {
    return wxString::From8BitData(mBuffer.str8(), mBuffer.getFillSize());
 }
-
-
-
-bool VST3Wrapper::FetchSettings(VST3EffectSettings& vst3Settings) const
```

Comment:
The meanings of `FetchSettings` and `StoreSettings` got reversed, and are now inconsistent with terminology in AudioUnits.  There, `FetchSettings` is fetching state *from* an AudioUnit handle *into* an EffectSettings package of data.  `StoreSettings` does the opposite.